profile/manifests/application/builder.pp

```diff
@@ -52,6 +52,11 @@
     managehome => true,
     gid        => $group,
     before     => Class['profile::openstack::openrc']
+  } ->
+  file { "/home/${user}/build_scripts":
+    ensure => directory,
+    owner  => $user,
+    group  => $group
   }
 
   create_resources('profile::application::builder::jobs', hiera_hash('profile::application::builder::images', {}))
```

Comment:
Vil anbefale å ha require på File["/home/${user}/build_scripts"] før du kjører jobs. Enten for hele create_resources eller for hver file i jobs.pp.